libraries/helpers.rb

```diff
@@ -151,14 +151,14 @@ def dhcpd_lease_file(ip_version)
 
       def dhcpd_lease_file_options
         case node['platform_family']
-        when 'amazon', 'centos', 'fedora', 'rhel'
+        when 'amazon' 'fedora', 'rhel'
           {
             'owner' => 'dhcpd',
             'group' => 'dhcpd',
             'mode' => '0644',
             'action' => :create_if_missing,
           }
-        when 'ubuntu'
+        when 'debian'
```

Comment:
This one can get nuked entirely

Reply:
That's fat fingers on my part, I've fixed it.